app/main.py

```diff
@@ -1 +1,38 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100
+    ) -> None:
+        self.name = name
+        self.hidden = False
+        self.health = health
+        self.alive.append(self)
+
+    def __repr__(
+            self
+    ) -> str:
+        return f"{{Name: {self.name}, Health: " \
+               f"{self.health}, Hidden: {self.hidden}}}"
+
+
+class Herbivore(Animal):
+
+    def hide(
+            self
+    ) -> None:
+        self.hidden = not self.hidden
+
+
+class Carnivore(Animal):
+
+    @staticmethod
+    def bite(
```

Comment:
you need to transfer each argument from a new line if all the arguments do not fit in 1 line, otherwise you don’t need to do this